scripts/Tools/Makefile

```diff
@@ -73,6 +73,8 @@ ifeq ($(strip $(USE_KOKKOS)), TRUE)
   USE_CXX = TRUE
 endif
 
```

Comment:
Only a few changes outside of the new config/ufs directory.  This one allows you to set SLIBS from buildlib